### PR TITLE
[CHF-568] Health Check keen-home-smart-vent

### PR DESCRIPTION
**Pull request by pchomal**
Health Checked keen-home-smart-vent with Readme

### CODE REVIEW

**Comment by pchomal** [comment]
@jackchi @ShunmugaSundar 

Please review & merge

File: devicetypes/keen-home/keen-home-smart-vent.src/keen-home-smart-vent.groovy [hunk 1]
```diff
@@ -510,5 +533,5 @@ def configure() {
         // "send 0x${device.deviceNetworkId} 1 1", "delay 1500",
     ]
 
-    return configCmds + refresh()
+    return configCmds + zigbee.onOffConfig(0, 300) + refresh()
```
Comment (jackchi) [inline devicetypes/keen-home/keen-home-smart-vent.src/keen-home-smart-vent.groovy]: The `5 minutes` reporting interval was a requirement set by VD (TV-Project). We no longer have these restraints. Let's use the default reporting `onOffConfig()` interval specified in: 
https://github.com/PhysicalGraph/appengine-zigbee/blob/master/src/main/groovy/physicalgraph/zigbee/Zigbee.groovy#L507
Which is `10min`.

**Review by jackchi — CHANGES_REQUESTED** [review]

**Comment by pchomal** [comment]
@jackchi 

I made changes for  keen home smart vent, removing the explicit setting of max report time as you said.
Then while going through its events saw that the temperature & pressure (especially) cluster are reporting CHANGES `every minute`.
It doesn't seem to be a reporting device, but one which sends status change events regularly.
So how should this be handled? And what should be the checkInterval for this case?

**Comment by jackchi** [comment]
Can you point out the line where it is sending a report for temp/pressure every minute? I only see the `bind` command.

File: devicetypes/keen-home/keen-home-smart-vent.src/keen-home-smart-vent.groovy [hunk 2]
```diff
@@ -466,15 +477,27 @@ def refresh() {
     getBattery()
 }
 
+/**
+ * PING is used by Device-Watch in attempt to reach the Device
+ * */
+def ping() {
+    return zigbee.onOffRefresh()
```
Comment (jackchi) [inline devicetypes/keen-home/keen-home-smart-vent.src/keen-home-smart-vent.groovy]: We can call `refresh()` here.

**Review by jackchi — CHANGES_REQUESTED** [review]
1. Change to default onOffConfig() 
2. Implement `ping` with `refresh`
3. Remove 2 different `checkInterval` in parse()
4. `checkInterval` based on default onOffConfig() = 10 *2 + 2=  22min

**Comment by pchomal** [comment]
@jackchi 
Made the requested changes.

File: devicetypes/keen-home/keen-home-smart-vent.src/keen-home-smart-vent.groovy [hunk 3]
```diff
@@ -466,15 +465,27 @@ def refresh() {
...
+    // Device-Watch allows 2 check-in misses from device + ping (plus 1 min lag time)
+    // enrolls with default periodic reporting until newer 5 min interval is confirmed
+    sendEvent(name: "checkInterval", value: 2 * 10 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
+
     // get ZigBee ID by hidden tile because that's the only way we can do it
     setZigBeeIdTile()
 
     def configCmds = [
         // bind reporting clusters to hub
-        "zdo bind 0x${device.deviceNetworkId} 1 1 0x0006 {${device.zigbeeId}} {}", "delay 500",
```
Comment (jackchi) [inline devicetypes/keen-home/keen-home-smart-vent.src/keen-home-smart-vent.groovy:477]: Is this bind cluster cmd suppose to be deleted?

Comment (pchomal) [reply devicetypes/keen-home/keen-home-smart-vent.src/keen-home-smart-vent.groovy:477]: This bind cluster corresponds to onOff cluster.
So as I'm using the wrapper, `zigbee.onOffConfig()`, so deleted that bind cluster cmd.

**Review by jackchi — APPROVED** [review]

**Comment by jackchi** [comment]
👍

**Pull request merged**